### ISSUE TITLE
detect and ignore unmounted systemd automount.

### ISSUE THREAD

**Issue by memchr** [muesli/duf#250]
When accessing a mount point specified with the x-systemd.autmount option in the fstab file, it is mounted on demand by the [systemd.automount](https://www.freedesktop.org/software/systemd/man/systemd.automount.html) feature.

Currently, `duf` will attempt to access an unmounted systemd automount point, resulting in a noticeable delay, and in the worst case, if the drive is offline, duf will hang indefinitely because the automount timeout is disabled by systemd by default.

```
systemd[1]: media-backups.automount: Got automount request for /media/backups, triggered by 741148 (duf)
```

Should it be a reasonable default for `duf` to ignore all unmounted systemd automounts?

**Comment by lingfish** [comment]
Agreed. `df` does the right thing. But I guess because this also does `du`, `du` would normally (obviously) trigger a mount. 

So `duf` should behave like `df`, and then only do `du` based on df behaviour.

**Comment by lingfish** [comment]
> Indeed, that sounds sensible.

Cool. Will you do it, or would you be wanting a PR?
